scripts/ubuntu14-leapfrog.sh

```diff
@@ -85,6 +85,11 @@ else
   log "rpc-preflight-check" "skipped"
 fi
 
+# Glance cache cleanup
+pushd /opt/rpc-upgrades/playbooks/
+    openstack-ansible glance-cache-cleanup.yml
+pop
```

Comment:
Resolved the conflict, but this should be popd

Reply:
Yeah, fixing it now